src/components/Menu/Menu.test.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { render, waitFor } from '@testing-library/react'
+
+import { category, divider, group, item } from './Menu.mocks'
+import { Menu } from '.'
+import { Modes } from './Menu.types'
+
+const { Vertical } = Modes
+
+describe('Menu Component', () => {
+  const originalErr = global.console.error.bind(global.console.error)
```

Comment:
Could you please add a comment to explain the reason behind this warning suppression?